<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CBAMainWidget</class>
 <widget class="QMainWindow" name="CBAMainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>即时通讯</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="CBALoginWidget" name="loginWidget"/>
      <widget class="CMainPageWidget" name="mainWidget"/>
      <widget class="CSettingsWidget" name="settingsWidget"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CSettingsWidget</class>
   <extends>QWidget</extends>
   <header>CBAClient/settings/CSettingsWidget.h</header>
   <container>1</container>
  </customwidget>
<customwidget>
 <class>CBALoginWidget</class>
 <extends>QWidget</extends>
 <header>CBAClient/login/CBALoginWidget.h</header>
 <container>1</container>
</customwidget>
  <customwidget>
   <class>CMainPageWidget</class>
   <extends>QWidget</extends>
   <header>CBAClient/main/CMainPageWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
